README.adoc

```diff
@@ -28,11 +28,13 @@ Learn how to develop tests for your microservices using the Arquillian Liberty M
 == What you'll learn
 You will learn how to develop tests for your microservices using the https://github.com/OpenLiberty/liberty-arquillian/tree/master/liberty-managed[Arquillian Liberty Managed container] and JUnit with Maven in Open Liberty. You will also learn how to take advantage of features in the Liberty plugin that simplify the process of managing Arquillian dependencies and setting up your managed container.
 
-http://arquillian.org/[Arquillian] is a testing platform to develop automated functional, integration and acceptance tests for your Java applications. Arquillian tests are very similar to the common unit tests, however, Arquillian sets up the test environment and is able to handle the application server life cycle for you, so you can focus on writing tests. Arquillian supports both JUnit and TestNG as the test runner. This guide will use the JUnit test runner. The Arquillian JUnit test has three components, `@RunWith(Arquillian.class)`, `@Deployment` and `@Test`. Once you have these three components setup, you'll have everything you need to test your microservices. You will learn more about these components in the test class.
+http://arquillian.org/[Arquillian] is a testing platform to develop automated functional, integration and acceptance tests for your Java applications.
 
-Arquillian allows different application servers to implement their own embedded, managed and remote containers. This guide will use the Arquillian Liberty Managed container, which is an adaptor to run Arquillian tests with Liberty. It's capable of managing the local Liberty server operations, packaging the tests as a `WAR` to be deployed to the Liberty server and then running the tests on the server.
+Arquillian tests looks similar to the common unit tests. The benefit of Arquillian test is that it sets up the test environment and is able to handle the application server life cycle for you, so you can focus on writing tests. Arquillian supports both JUnit and TestNG as the test runner. This guide will use the JUnit test runner. An Arquillian JUnit test has three core components, `@RunWith(Arquillian.class)`, `@Deployment` and `@Test`. Once you have these three components setup, you'll have everything you need to test your microservices. You will learn more about these components in the test class.
```

Comment:
Grammar issue - "Arquillian tests looks similar to the common unit tests."

Say "Arquillian tests look similar to common unit tests."

Reply:
Fixed